affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt

```diff
@@ -134,6 +136,9 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
         val task = rootProject.tasks.register(taskType.commandByImpact).get()
         task.group = groupName
         task.description = taskType.taskDescription
+        if (GradleVersion.current() >= GradleVersion.version("7.4")) {
```

Comment:
nit: We can transform it to const in companion object

Reply:
You mean the `7.4`? Seems a bit unnecessary since `GradleVersion.version(GRADLE_VERSION_7_4)` is less readable too IMO